public_html/wp-content/mu-plugins/jetpack-tweaks/css-sanitization.php

```diff
@@ -244,12 +244,19 @@ function get_custom_css_properties_safelist() {
 		array(
 			'aspect-ratio',
 			'background-blend-mode',
+			'box-orient',
 			'isolation',
+			'line-clamp',
 			'mask',
 			'mix-blend-mode',
+			'overscroll-behavior',
+			'place-content',
 			'shape-image-threshold',
 			'shape-outside',
+			'text-decoration-thickness',
 			'text-orientation',
+			'text-stroke',
+			'text-fill',
```

Comment:
These are [also non-standard](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke), and appear to be supported only with a prefix. Should only the prefixed properties be added?

Reply:
Ah the magic of devtools, missed that! I've pushed a commit with the `-webkit-*` prefix for the three properties that'll need 'em.